rfc-013-code-repositories.md

```diff
@@ -0,0 +1,77 @@
+# Code repository standards
+
+## Summary
+
+A set of standards to be used for projects that will be shared within Made Tech or the greater technology community. These are projects hosted on GitHub, Bitbucket, Gitlab or similar public source control management.
+
+## Problem
+
+Made Tech code repositories (to be referred to as repos herein) do not have a common look and feel.
+
+## Visibility
+
+The visibility of your project **SHOULD** mean the following:
```

Comment:
I'd prefer to flip this around.

- You **MUST** set your repo to **Public** unless you have a reason not to:
- You **MUST** set your repo to **Private to the Customers organisation** if the customer owns the Intellectual Property and has a public source control management platform account
- You **MUST** set your repo to **Private to the Made Tech organisation** if the customer owns the Intellectual Property and has a public source control management platform account

Can we also recommend that all repositories should come with `README.md` and `LICENSE`?

Reply:
Thanks for that, @craigjbass had a similar view. I agree with you both open by default.